Sources/BraintreeCard/CreditCardGraphQLBody.swift

```diff
@@ -0,0 +1,178 @@
+// swiftlint:disable nesting
+
+import Foundation
+
+struct CreditCardGraphQLBody: Encodable {
```

Comment:
Can we move this swiftlint disable here instead:
```suggestion
// swiftlint:disable nesting
struct CreditCardGraphQLBody: Encodable {
```